### PR TITLE
[Explorer] set up new home page

### PR DESCRIPTION
**Pull request by zihan-aptos**
Screen Shot 2022-09-21 at 3 07 23 PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 # misc
 .DS_Store
+.env
```
Comment (msavir) [inline .gitignore:20]: wasn't that committed in a previous PR?

File: src/api/hooks/useGetCoinSupplyLimit.tsx [hunk 2]
```diff
@@ -29,9 +32,25 @@ interface AggregatorData {
 }
 
 async function fetchTotalSupply(
-  aggregatorData: AggregatorData,
+  data: Types.MoveResource[],
   state: GlobalState,
 ): Promise<number | null> {
+  const coinInfo = data.find(
```
Comment (msavir) [inline src/api/hooks/useGetCoinSupplyLimit.tsx:38]: why not use `useGetAccountResource`? https://github.com/aptos-labs/explorer/blob/main/src/api/hooks/useGetAccountResource.ts

File: src/pages/LandingPage/NetworkInfo/MetricCard.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+type MetricCardProps = {
+  data: string;
+  label: string;
+};
+
+export default function MetricCard({data, label}: MetricCardProps) {
+  return (
+    <Card>
+      <Stack alignItems="flex-end" spacing={2.5} marginTop={2}>
+        <Typography sx={{fontSize: 17, fontWeight: 700}}>{data}</Typography>
```
Comment (msavir) [inline src/pages/LandingPage/NetworkInfo/MetricCard.tsx:15]:
```suggestion
        <Typography fontSize={17} fontWeight={700}>{data}</Typography>
```

**Review by msavir — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (zihan-aptos) [reply .gitignore:20]: I'll rebase

File: src/api/hooks/useGetCoinSupplyLimit.tsx (hunk 2 above)
Comment (zihan-aptos) [reply src/api/hooks/useGetCoinSupplyLimit.tsx:38]: good point

**Pull request merged**